documentation/asciidoc/computers/camera/rpicam_vid.adoc

```diff
@@ -1,120 +1,160 @@
-
-NOTE: Recent versions of VLC seem to have problems with playback of H.264 streams. We recommend using `ffplay` for playback using the above commands until these issues have been resolved.
+To suppress the preview window on the server, use the `--nopreview` (`-n`) flag. You can also use the `--inline` flag to force stream header information into every intra frame, which helps clients understand the stream if they miss the beginning.
 
 ==== High framerate capture
 
-Using `rpicam-vid` to capture high framerate video (generally anything over 60fps) while minimising frame drops requires a few considerations.
+To minimise frame drops for high framerate (> 60fps) video, try the following configuration tweaks:
 
-* The https://en.wikipedia.org/wiki/Advanced_Video_Coding#Levels[H.264 target level] must be set to 4.2 with the `--level 4.2` argument.
-* Software colour denoise processing must be turned off with the `--denoise cdn_off` argument.
-* For rates over 100 fps, disabling the display window with the `-n` option would free up some additional CPU cycles to help avoid frame drops.
-* It is advisable to set `force_turbo=1` in xref:../computers/config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] to ensure the CPU clock does not get throttled during the video capture. See xref:config_txt.adoc#force_turbo[the `force_turbo` documentation] for further details.
+* Set the https://en.wikipedia.org/wiki/Advanced_Video_Coding#Levels[H.264 target level] to 4.2 with `--level 4.2`.
+* Turn off software colour denoise processing with the `--denoise cdn_off` argument.
+* Disable the display window with the `--nopreview` (`-n`) flag to free up some additional CPU cycles.
+* Set `force_turbo=1` in xref:../computers/config_txt.adoc#what-is-config-txt[`/boot/firmware/config.txt`] to ensure that the CPU clock does not throttle during video capture. For more information, see xref:config_txt.adoc#force_turbo[the `force_turbo` documentation].
 * Adjust the ISP output resolution with `--width 1280 --height 720` or something even lower to achieve your framerate target.
-* On a Pi 4, you can overclock the GPU to improve performance by adding `gpu_freq=550` or higher in `/boot/firmware/config.txt`.  See xref:config_txt.adoc#overclocking[the overclocking documentation] for further details.
+* On Raspberry Pi 4, you can overclock the GPU to improve performance by adding `gpu_freq=550` or higher in `/boot/firmware/config.txt`.  See xref:config_txt.adoc#overclocking[the overclocking documentation] for further details.
 
-An example command for 1280×720 120fps video encode would be:
+The following command demonstrates how you might achieve 1280×720 120fps video:
 
-[,bash]
+[source,console]
 ----
-rpicam-vid --level 4.2 --framerate 120 --width 1280 --height 720 --save-pts timestamp.pts -o video.264 -t 10000 --denoise cdn_off -n
+$ rpicam-vid --level 4.2 --framerate 120 --width 1280 --height 720 --save-pts timestamp.pts -o video.264 -t 10000 --denoise cdn_off -n
```

Comment:
As you mentioned earlier, `--save-pts` doesn't work on a Pi 5, so it might be useful to include a Pi 5-compatible version of this command too?
I've successfully used
```console
rpicam-vid  --level 4.2  --framerate 120 --width 1280 --height 720 -o video.mkv -t 10000 --denoise cdn_off -n
```
to capture high-framerate video on a Pi 5.